assignments/objects.js

```diff
@@ -18,27 +18,68 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const mitzi = { 
+  id: 1, 
+  first-name: "Mitzi", 
```

Comment:
make sure if you want to put special characters like `-` as a key in your object, you wrap your key with `""` or else this will throw an error. I would change this to `firstName`